DEVELOPER_SETUP.md

```diff
@@ -0,0 +1,415 @@
+# VirtoCommerce Platform - Developer Setup Guide
+
+## 🚀 Quick Start for Developers
+
+This guide addresses common setup issues and provides step-by-step instructions to get VirtoCommerce Platform running locally.
+
+## ⚠️ Common Issues & Solutions
+
+### Issue 1: "Unable to run your project" (Docker Compose Error)
+**Problem**: Running `dotnet run` from wrong directory shows Docker Compose error  
+**Cause**: VirtoCommerce has multiple projects; running from root triggers Docker Compose mode  
+**Solution**: Always run from the web project directory:
+
+```bash
+# ❌ Wrong (from root directory - triggers Docker Compose error)
+D:\work\vc-platform> dotnet run
+
+# ✅ Correct (from web project directory)  
+D:\work\vc-platform\src\VirtoCommerce.Platform.Web> dotnet run
+```
+
+### Issue 2: PowerShell Syntax Errors on Windows
+**Problem**: Using bash syntax (`&&`) in PowerShell  
+**Solution**: Use PowerShell syntax:
+
+```powershell
+# ❌ Wrong (bash syntax)
+cd src\VirtoCommerce.Platform.Web && dotnet run
+
+# ✅ Correct (PowerShell syntax)
+cd src\VirtoCommerce.Platform.Web; dotnet run
+```
+
+### Issue 3: Port Conflicts
+**Problem**: "Address already in use" errors on startup  
+**Solution**: Kill existing processes:
+
+```powershell
+# Check what's using the port
+netstat -ano | findstr ":10645"
+
+# Kill the specific process (replace XXXX with Process ID)
+Stop-Process -Id XXXX -Force
+
+# Or kill all dotnet processes
+Stop-Process -Name "dotnet" -Force -ErrorAction SilentlyContinue
+```
+
+## 🛠️ Step-by-Step Setup
+
+### Prerequisites
+- **.NET 8 SDK** (Download from Microsoft)
+- **Node.js 18+** (For frontend development)
+- **SQL Server** (Local installation or Docker)
+- **Git** (For cloning repositories)
+
+### Backend Setup
+
+1. **Clone and navigate to the CORRECT directory:**
+```bash
+git clone https://github.com/VirtoCommerce/vc-platform.git
+cd vc-platform
+
+# ⚠️ CRITICAL: Navigate to the web project directory
+cd src/VirtoCommerce.Platform.Web
+```
+
+2. **Setup SQL Server:**
+
+**Option A: Docker (Recommended)**
+```bash
+docker run -d --name vc-sqlserver \
+  -e "ACCEPT_EULA=Y" \
+  -e "MSSQL_PID=Express" \
+  -e "SA_PASSWORD=Pass@word" \
+  -p 1433:1433 \
+  mcr.microsoft.com/mssql/server:latest
+```
+
+**Option B: Local SQL Server**
+Update `appsettings.Development.json`:
```

Comment:
Changing platform files is not recommended. Instead, you should use user secrets or environment variables.